src/CommunityToolkit.Maui.MediaElement/Extensions/PageExtensions.android.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Android.Views;
+using Activity = Android.App.Activity;
+
+namespace CommunityToolkit.Maui.Extensions;
+partial class PageExtensions
+{
+	public record struct CurrentPlatformActivity()
```

Comment:
we will need to change the name, since it returns more stuff than the current activity

Reply:
I have no idea what we should name it. Any suggestion?